topcoffea/modules/histEFT.py

```diff
@@ -318,6 +317,56 @@ def __reduce__(self):
             ),
         )
 
+    def make_scalings(self,h,ch,p):
+            scalingsbins = h[{
+              'channel': ch,
```

Comment:
This will break unless the histogram has a 'channel' and 'process' axis.  It would be better just to have `make_scalings` return the scaling for whatever histogram it's called on (including all axes) and have the caller restrict it to subsets by explicitly asking for that.  (E.g. call it like `s = h[{'channel':ch,'process':p,'systematic':'nominal'}.make_scalings()`).  Note, in my suggested version, `make_scalings` has no arguments to pass in.